pytket/pytket/circuit/display/__init__.py

```diff
@@ -73,3 +74,20 @@ def render_circuit_jupyter(
     :param circuit: the circuit to render.
     """
     render_circuit_as_html(circuit, True)
+
+
+def view_browser(circuit: Union[Dict[str, Union[str, float, dict]], Circuit]) -> None:
```

Comment:
Could we make the `new` parameter to `webbroser.open()`, and the time passed to `sleep()`, into optional arguments (with the defaults you have)?